### PR TITLE
fix travis and lint

### PR DESCRIPTION
**Pull request by ghost**
much sexier

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,3326 +0,0 @@
-{
```
Comment (wolfy1339) [inline package-lock.json:1]: Why are you removing this? Can you explain this change.

File: src/index.js [hunk 2]
```diff
@@ -2,81 +2,37 @@ const axios = require('axios');
 const helpers = require('./helpers');
 
 const headers = {
-	'Accept': 'application/vnd.codebottle.v1+json'
+  Accept: 'application/vnd.codebottle.v1+json',
 };
 
 module.exports = {
-	search: (keywords, language) => {
```
Comment (wolfy1339) [inline src/index.js:9]: What program and config did you use for linting this?
I don't think applying what you think is 'sexier' is appropriate. This project already has a style, albeit not dicumented.

File: package-lock.json (hunk 1 above)
Comment (ghost) [reply package-lock.json:1]: Package lock is malware unless you are making unpublished to registry project like bot

Comment (wolfy1339) [reply package-lock.json:1]: And why is it 'malware'?

File: src/index.js (hunk 2 above)
Comment (ghost) [reply src/index.js:9]: It’s eslint and v0id config 

I replaced it with new style as it’s easier to maintain and read
Also it complies with config rules

File: package-lock.json (hunk 1 above)
Comment (ghost) [reply package-lock.json:1]: locks user out from easy version switching for library used packages

**Comment by MicroDroid** [comment]
No.

**Pull request closed without merging**